serve/graph/transaction.resolvers.go

```diff
@@ -0,0 +1,21 @@
+package graph
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.45
+
+import (
+	"context"
+
+	"github.com/gnolang/tx-indexer/serve/graph/model"
+)
+
+// ResponseData is the resolver for the response_data field.
+func (r *transactionResolver) ResponseData(ctx context.Context, obj *model.Transaction) (string, error) {
+	return string(obj.ResponseData()), nil
+}
```

Comment:
the graphql framework generated this because types are not matching. You are returning []byte from the model, but requesting !String from the schema.

Reply:
I fixed it in this commit. 
https://github.com/gnolang/tx-indexer/pull/32/commits/8cf8f5ccae08afa56bbde31f570356ff3b4a23cb